packages/fxa-settings/src/components/Settings/AccountRecoveryWizard/index.tsx

```diff
@@ -0,0 +1,133 @@
+    We have filler components hardcoded in here (the Wizard pattern seems too simple to justify making it into a reusable component), but we can replace these as we create the real views.
+    Each step should be a separate component.
+  */
+  return (
+    <>
+      {
+        // Create an account recovery key in case you forget your password
+        currentStep === 1 && (
+          <FlowContainer
+            title="Account recovery key"
+            {...{ currentStep, setCurrentStep }}
+            onBackButtonClick={() => {
+              window.history.back();
+            }}
+          >
+            <ProgressBar currentStep={1} numberOfSteps={4} />
+            <h2 className="font-bold text-xl mb-2">
+              Create an account recovery key in case you forget your password
+            </h2>
+            <p>First step view</p>
+            <button
+              className="cta-primary cta-base-p mx-2 flex-1"
+              type="button"
+              onClick={() => {
+                setCurrentStep(currentStep + 1);
+              }}
+            >
+              Next view
+            </button>
+          </FlowContainer>
+        )
+      }
+      {
+        // Enter your password again to get started
+        currentStep === 2 && (
+          <FlowContainer
+            title="Account recovery key"
+            {...{ currentStep, setCurrentStep }}
+            onBackButtonClick={() => {
+              setCurrentStep(currentStep - 1);
+            }}
+          >
+            {' '}
+            <ProgressBar currentStep={2} numberOfSteps={4} />
+            <h2 className="font-bold text-xl mb-2">
+              Enter your password again to get started
+            </h2>
+            <button
+              className="cta-primary cta-base-p mx-2 flex-1"
+              type="button"
+              onClick={() => {
+                setCurrentStep(currentStep + 1);
+              }}
+            >
+              Next view
+            </button>
+          </FlowContainer>
+        )
+      }
+      {
+        // Account recovery key generated
+        currentStep === 3 && (
+          <FlowContainer
+            title="Account recovery key"
+            {...{ currentStep, setCurrentStep }}
+            onBackButtonClick={() => {
+              setCurrentStep(currentStep - 1);
+            }}
+          >
+            <ProgressBar currentStep={3} numberOfSteps={4} />
+            <h2 className="font-bold text-xl mb-2">
+              Account recovery key generated
+            </h2>
+            <button
+              className="cta-primary cta-base-p mx-2 flex-1"
+              type="button"
+              onClick={() => {
+                setCurrentStep(currentStep + 1);
+              }}
+            >
+              Next view
+            </button>
+          </FlowContainer>
```

Comment:
A lot of the code in these steps looks pretty boilerplate. Is it worth extracting out into some higher level component that wraps the progressbar, h2, and button? I think the only customized stuff is back button and next button scripts, h2+button text, and the current+total step numbers. Plus reduces the chances of the styling and markup getting out of sync.

Reply:
These are gonna be replaced with individual components which share a lot of characteristics -- we can comment them out, or remove them entirely (instead of removing them as we complete the individual steps). I just left them in so that it was clear where those views would be, since we'll be doing those asynchronously